### ISSUE TITLE
Stream handler invoked with the wrong structure

### ISSUE THREAD

**Issue by jkarsrud** [hicksy/arc-plugin-sandbox-stream#3]
First of all; thank you for a great plugin! Paired with a small plugin that updates the table with a `StreamSpecification` it's pretty much flawless! 

However, I have found a small issue; when the plugin invokes the stream handler defined in `.arc`, it invokes it by passing `data.Records` instead of `data`. This can be [seen here](https://github.com/hicksy/arc-plugin-sandbox-stream/blob/d31a5ea479ebab8292c7fdd9059c3b6055fa48b6/index.js#L41-L47). This means that we need to do some extra checks in our handler to identify whether it was locally invoked or not. This might not be the intention, so I thought I'd drop an issue :) 

Let me know if you'd like some steps to reproduce!

**Comment by hicksy** [comment]
Yep you're right, will make complete sense to pass `data` to the invoked function.

Just back from holiday so I'll take a look into this early next week.

Appreciate your kind words, thanks  🙏

**Comment by lpsinger** [comment]
> Just back from holiday so I'll take a look into this early next week.

Will you be able to fix this any time soon?

**Comment by hicksy** [comment]
😬

So sorry, life got in the way. 

Thanks for your patience. Will update later tonight / tomorrow

**Comment by hicksy** [comment]
Version 1.1.0 has been published to npm with change.  

Thanks 👍
